src/index.d.ts

```diff
@@ -86,6 +262,21 @@ declare namespace admin.app {
     messaging(): admin.messaging.Messaging;
     projectManagement(): admin.projectManagement.ProjectManagement;
     storage(): admin.storage.Storage;
+    /**
```

Comment:
Couple of developers have pointed out that we should explain this method a bit more in our documentation (see #489 for example). Can we add a line here indicating that when running the SDK locally, this method must be called to ensure graceful termination of the process?

Reply:
Added more detail poached from your comments.  PTAL.

Reply:
Hi Eric. If you've made any changes since my last review, you need to commit and push them.

Reply:
Can you see the changes now Hiranya?  I did manage to merge and push :)